sections/method/PageRank.tex

```diff
@@ -1,4 +1,14 @@
 \section{PageRank}\label{sec:pagerank}
+\gls{pr} is a recursive ranking algorithm based on the random surfer model, where a number of surfers would take $s$ amount of steps across a graph. 
```

Comment:
```suggestion
\gls{pr} is a recursive probabilistic ranking algorithm, used to rank nodes in a graph. It is based on the random surfer model, where a number of surfers would take $s$ amount of steps across a graph. 
```

Reply:
\gls{pr} is a recursive ranking algorithm, used to rank nodes in a graph. 
It is based on the random surfer model, where several surfers would take $s$ amount of steps across a graph.